cl/search/templates/includes/search_result.html

```diff
@@ -27,7 +27,9 @@ <h3 class="bottom serif">
       <a href="{{ result.absolute_url }}?{{ request.META.QUERY_STRING }}"
          class="visitable">
       {{ result.solr_highlights.caseName.0|safe }}
-      ({% if result.court_id != 'scotus' %}{{ result.solr_highlights.court_citation_string.0|nbsp|safe }}&nbsp;{% endif %}{% if type == SEARCH_TYPES.OPINION or type_override == SEARCH_TYPES.OPINION %}{{ result.dateFiled|date:"Y" }}{% else %}{{ result.dateArgued|date:"Y" }}{% endif %})
+        {% if maine != True %}
+           ({% if result.court_id != 'scotus' %}{{ result.solr_highlights.court_citation_string.0|nbsp|safe }}&nbsp;{% endif %}{% if type == SEARCH_TYPES.OPINION or type_override == SEARCH_TYPES.OPINION %}{{ result.dateFiled|date:"Y" }}{% else %}{{ result.dateArgued|date:"Y" }}{% endif %})
```

Comment:
Let me see, I think that might have bad downstream effects

Reply:
I think this is what the answer should be 

``` 
{% if request.path != "/court/me/" %}
   ({% if result.court_id != 'scotus' %}{{ result.solr_highlights.court_citation_string.0|nbsp|safe }}&nbsp;{% endif %}
      {% if type == SEARCH_TYPES.OPINION or type_override == SEARCH_TYPES.OPINION %}
     {{ result.dateFiled|date:"Y" }}{% else %}{{ result.dateArgued|date:"Y" }}{% endif %})
{% endif %}
```

I'm trying to avoid affecting the search results page.  The example you provided doesnt really work because the scotus stuff happens inside what we want or dont